services/server/test/testcontracts/Storage/Storage.json

```diff
@@ -626,32 +626,22 @@
       "name": "#utility.yul"
     }
   ],
-  "sourceMap": "141:356:8:-:0;;;;;;;;;;;;;;;;;;;",
-  "deployedSourceMap": "141:356:8:-:0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;416:79;;;:::i;:::-;;;;;;;:::i;:::-;;;;;;;;271:64;;;;;;;;;;;;;:::i;:::-;;:::i;:::-;;416:79;457:7;482:6;;475:13;;416:79;:::o;271:64::-;325:3;316:6;:12;;;;271:64;:::o;7:139:9:-;53:5;91:6;78:20;69:29;;107:33;134:5;107:33;:::i;:::-;59:87;;;;:::o;152:262::-;211:6;260:2;248:9;239:7;235:23;231:32;228:2;;;276:1;273;266:12;228:2;319:1;344:53;389:7;380:6;369:9;365:22;344:53;:::i;:::-;334:63;;290:117;218:196;;;;:::o;420:118::-;507:24;525:5;507:24;:::i;:::-;502:3;495:37;485:53;;:::o;544:222::-;637:4;675:2;664:9;660:18;652:26;;688:71;756:1;745:9;741:17;732:6;688:71;:::i;:::-;642:124;;;;:::o;772:77::-;809:7;838:5;827:16;;817:32;;;:::o;855:122::-;928:24;946:5;928:24;:::i;:::-;921:5;918:35;908:2;;967:1;964;957:12;908:2;898:79;:::o",
+  "sourceMap": "141:356:0:-:0;;;;;;;;;;;;;;;;;;;",
```

Comment:
Why did this become 0?

Reply:
`141:356:8` -> `141:356:0`

Reply:
Good question. Tbh, I just recompiled myself and added the source map there because I had a failing test. I don't know why the server and my compilation produce a different source map than the original compilation. Everything else just matches. Any idea how I can get the same source map as before?

Reply:
Mmh I don't have an answer right now, maybe @kuzdogan knows it?

Reply:
No idea, I tried to compile the same file with different compilers but I didn't get an inconsistent result. The new value is the correct one.